test/test_helper.exs

```diff
@@ -1 +1,2 @@
+{:ok, _apps} = Application.ensure_all_started([:plug])
```

Comment:
`:plug` doesn't need to be started, actually, since the tests are not using `Plug.Upload`